res/autoload.template

```diff
@@ -0,0 +1,14 @@
+<?php
+$vendor = '/usr/share/php';
```

Comment:
Instead of setting this variable into the global namespace, can we "hard code" it in the `require_once` below?  I say "hard code" because I see it as a placeholder in the string that this pkg's spec file would replace with the actual real value (obviously only if this template file is provided by this pkg in it's RPM like previous question).